bot/iiwb/bot/app.py

```diff
@@ -0,0 +1,41 @@
+from iiwb.client.iiwb import Reverse, __title__, __version__, __codename__, __fullcodename__
+from discord.ext import commands
+from discord.utils import get
+import discord
+from discord import app_commands
+from discord import __version__ as dversion
+import asyncio
+import sys
+
+class Bot(Reverse):
```

Comment:
I don't think iiwb will host several instances of bots (so defining Reverse could be directly done in this class) but open to discussion